swap_meet/vendor.py

```diff
@@ -1,2 +1,62 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, inventory_item):
+        self.inventory.append(inventory_item)
+        return inventory_item
+
+    def remove(self, inventory_item):
+        if inventory_item in self.inventory:
+            self.inventory.remove(inventory_item)
+            return inventory_item
+        else:
+            return False
```

Comment:
Notice how the else clause only has one line of code? `return False` doesn't have to live in an else clause. In fact, we can flip the conditional logic so that the primary logic of this function is emphasized and reduces the lines of code in this function by 1. 

```suggestion
    def remove(self, inventory_item):
        if inventory_item not in self.inventory:
            return False
        self.inventory.remove(inventory_item)
        return inventory_item
```